web_duplicate_visibility/README.rst

```diff
@@ -0,0 +1,82 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
you should move this to the `description` key in the manifest as 7.0 doesn't look at this file. This will avoid runbot's warning `WARNING 3150139-383-e4882b-all openerp.addons.base.module.module: module web_duplicate_visibility: description is empty` and make your branch green